src/modules/filters/components/FilterBar/FilterBar.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import useFelaEnhanced from 'hooks/useFelaEnhanced';
+import { Select, Form, Space } from 'antd';
+
+import uuid from 'react-uuid';
+
+import * as felaRules from './FilterBar.styles';
+import useFilters from '../../hooks/useFilters';
+const { Option } = Select;
+
+const FilterBar = () => {
+    const { styles } = useFelaEnhanced(felaRules);
+
+    const { appName, platform, deviceModel } = useFilters();
+    return (
+        <div className={styles.container}>
+            <Form className={styles.container} size="large" layout="vertical">
+                <Space size={'large'}>
+                    <Form.Item className={styles.formItem} label="App name" name="AppName">
```

Comment:
this labels should be stored in localizations files

Reply:
Done****